### PR TITLE
Latest round for the waffle github action

### PR DESCRIPTION
**Pull request by jimleroyer**
# Summary | Résumé

Latest round for the waffle github action. Current action failed to load the entry point with the docker image configuration and missed required Flask dependency. There are a few minor changes all around as well.

# Test instructions | Instructions pour tester la modification

I am basically testing the action right now [through this PR](https://github.com/cds-snc/notification-document-download-api/pull/51).

### CODE REVIEW

File: .github/actions/waffles/README.md [hunk 1]
```diff
@@ -44,8 +44,8 @@ None other than the log output and system exit code.
 
 uses: actions/waffles@v1
 with:
-  app-libs: '/Projects/cds/notification-document-download-api'
   app-loc: '/doc-api-env/Lib/site-packages'
+  app-libs: '/Projects/cds/notification-document-download-api'
   flask-mod: 'application'
   flask-prop: 'application'
   base-url: 'https://api.document.staging.notification.cdssandbox.xyz'
```
Comment (sastels) [inline .github/actions/waffles/README.md:51]: should have this block in ```'s

File: .github/actions/waffles/README.md [hunk 2]
```diff
@@ -44,8 +44,8 @@ None other than the log output and system exit code.
 
 uses: actions/waffles@v1
 with:
-  app-libs: '/Projects/cds/notification-document-download-api'
   app-loc: '/doc-api-env/Lib/site-packages'
+  app-libs: '/Projects/cds/notification-document-download-api'
```
Comment (sastels) [inline .github/actions/waffles/README.md:48]: `app-loc` should be '/Projects/cds/notification-document-download-api' and `app-libs` '/doc-api-env/Lib/site-packages', right?

**Review by sastels — APPROVED** [review]
Couple README suggestions, 🚀

File: .github/actions/waffles/README.md (hunk 2 above)
Comment (jimleroyer) [reply .github/actions/waffles/README.md:48]: Right, I changed that.

File: .github/actions/waffles/README.md (hunk 1 above)
Comment (jimleroyer) [reply .github/actions/waffles/README.md:51]: Done.

**Pull request merged**